internal/sentrylib/sentrylib.go

```
@@ -0,0 +1,26 @@
+package sentrylib
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/getsentry/sentry-go"
+	"github.com/xplorer-io/xplorers-bot-go/internal/config"
+	"github.com/xplorer-io/xplorers-bot-go/internal/ssm"
+)
+
+func InitSentry(env *config.Config) {
+	sentryDsn, getParamErr := ssm.GetSsmParameter(env.SentryDsnSsmPath)
+	if getParamErr != nil {
+		fmt.Printf("Unable to fetch sentry dsn: %s", getParamErr)
```

this should return error not print.
```
fmt.Errorf("Unable to fetch sentry dsn: %v", getParamErr)
```

---

If we respond to slack with anything other than 200 response, slack will retry upto 3 times as per the docs [here](https://api.slack.com/apis/connections/events-api#failure_limits:~:text=We%27ll%20knock%20knock%20knock%20on%20your%20server%27s%20door%2C%20retrying%20a%20failed%20request%20up%20to%203%20times%20in%20a%20gradually%20increasing%20timetable%3A). If we fail to process slack event, we should be handling it or be fixing the bug if any. 

We could however set the header `X-Slack-No-Retry: 1` to turn off retries as seen in this [document](https://api.slack.com/apis/connections/events-api#the-events-api__field-guide__error-handling__graceful-retries:~:text=know%20why%20yet.-,Turning%20retries%20off,-If%20your%20server), shall we return an error along with this header so slack does not retry?